kubernetes/deployment.yml

```diff
@@ -23,7 +23,13 @@ spec:
             path: config.yml
```

Comment:
For the record, this was me checking that we didn't need to worry about creating helm charts etc., because this isn't being deployed part of G8s